<launch>
   <include file="$(find rcvm_matrice)/launch/rcvm.launch"/>
	<node pkg="rcvm_matrice" type="indicate_object_right.sh" name="indicate_object"/>
</launch>

